syntax/model.js

```diff
@@ -39,340 +39,342 @@ import {
   blockName,
 } from "./blocks.js"
 
-/* Label */
-
-export var Label = function (value, cls) {
-  this.value = value
-  this.cls = cls || ""
-  this.el = null
-  this.height = 12
-  this.metrics = null
-  this.x = 0
+export class Label {
+  constructor(value, cls) {
+    this.value = value
+    this.cls = cls || ""
+    this.el = null
+    this.height = 12
+    this.metrics = null
+    this.x = 0
+  }
+
+  stringify() {
+    if (this.value === "<" || this.value === ">") return this.value
+    return this.value.replace(/([<>[\](){}])/g, "\\$1")
+  }
 }
 Label.prototype.isLabel = true
```

Comment:
It irks me that there isn't a nicer way of writing this. `static isLabel = true` doesn't work; and `get isLabel() { return true }` isn't strictly equivalent!

Reply:
Ah that totally works, nice!